domains/snek.json

```diff
@@ -0,0 +1,11 @@
+ 
+    {
+      "owner": {
+        "username": "sn3kdev",
+        "email": "snektg@gmail.com"
+      },
+      "record": {
+        "URL": ""odnako_zmeyka.t.me""
```

Comment:
Please remove the extra quotations and add `https://` infront of the link

Reply:
Had committed

Reply:
https://github.dev/sn3kdev/register/tree/snek-https